ios_list_data/ViewModel/ListItemViewModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ListItemViewModel.swift
+//  ios_list_data
+//
+//  Created by Nam on 13/12/2022.
+//
+
+import Foundation
+import Alamofire
+
+protocol ListItemDelegate {
+    func didLoad()
+}
+
+
+class ListItemViewModel {
+    
+    var apiHandler = APIHandler.sharedIntance
+    
+    var listArticle = [Article]()
+    
+    var delegate: ListItemDelegate?
```

Comment:
read more about retain cycle
var delegate is strong reference
cause of dead lock then object can not be release